### PR TITLE
create folder objects and solved prototyping exercises

### PR DESCRIPTION
**Pull request by alexadark**
I have created the new directory objects, with inside a prototyping folder, including the readme file, and solved the exercise

### CODE REVIEW

**Comment by Micheloncio** [comment]
👍

File: course/frontend/objects/prototyping/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (alexadark) [reply course/frontend/objects/prototyping/index.html]: There is already a solution in prototype-2, should I make a prototype-3 ?

**Comment by manuelbarzi** [comment]
REDO the PR from a custom branch containing ONLY the code corresponding to a specific challenge, but not a mix of many of them (code messes up, otherwise).

File: course/frontend/objects/prototyping/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (manuelbarzi) [reply course/frontend/objects/prototyping/index.html]: not necessarily. just put your code below the existing code, wrapped in a self-invoking function, and that will isolate it and run it safety ,)

**Pull request closed without merging**